modelon/impact/client/entities/experiment.py

```diff
@@ -214,6 +224,13 @@ def _get_info(self, cached: bool = True) -> Dict[str, Any]:
 
         return self._info
 
+    def get_size_info(self) -> ExperimentSizeInfo:
+        """The experiment size information."""
+        run_info = self._sal.workspace.experiment_get(
+            self._workspace_id, self._exp_id, size_info=True
+        )["run_info"]
+        return ExperimentSizeInfo(run_info.get("sizeInfo", {}))
```

Comment:
Why are you checking for empty dict here? It seems more like an exception of some kind?